memq-actor/src/main/java/io/appform/memq/retry/config/CountLimitedExponentialWaitRetryConfig.java

```diff
@@ -18,36 +21,34 @@ public class CountLimitedExponentialWaitRetryConfig extends RetryConfig {
 
     @Min(1)
     @Builder.Default
-    int maxAttempts = 1;
+    private int maxAttempts = 1;
 
-    @Min(1)
-    @Builder.Default
-    int delayInMillis = 1;
-
-    @Min(2)
+    @Valid
+    @NotNull
     @Builder.Default
-    int maxDelayInMillis = 100;
+    private Duration waitTime = Duration.ofMillis(10);
 
-    @Min(1)
+    @Valid
+    @NotNull
     @Builder.Default
-    int maxTimeInMillisBetweenRetries = 1;
+    private Duration maxWaitTime = Duration.ofMillis(1_000);
```

Comment:
if there is a dropwizard dependency here, use dropwizard duration. Othewise keep it to millis. The java time duration strings are not easy to understand and configure

Reply:
Moved to millis